src/protocols/secure_channel/CASESession.cpp

```diff
@@ -68,17 +67,17 @@ enum
     kTag_TBSData_ReceiverPubKey = 4,
 };
 
-enum
-{
-    kTag_Sigma1_InitiatorRandom    = 1,
-    kTag_Sigma1_InitiatorSessionId = 2,
-    kTag_Sigma1_DestinationId      = 3,
-    kTag_Sigma1_InitiatorEphPubKey = 4,
-    kTag_Sigma1_InitiatorMRPParams = 5,
-    kTag_Sigma1_ResumptionID       = 6,
-    kTag_Sigma1_InitiatorResumeMIC = 7,
-};
-
+// Sigma1 TLV Tags
+inline constexpr uint8_t kInitiatorRandomTag    = 1;
+inline constexpr uint8_t kInitiatorSessionIdTag = 2;
+inline constexpr uint8_t kDestinationIdTag      = 3;
+inline constexpr uint8_t kInitiatorPubKeyTag    = 4;
+inline constexpr uint8_t kInitiatorMRPParamsTag = 5;
+inline constexpr uint8_t kResumptionIDTag       = 6;
+inline constexpr uint8_t kResume1MICTag         = 7;
+
+// Sigma2 TLV Tags
+// TODO: either use constexpr, or enum for all
```

Comment:
how about we make things consistent for the new code and use enum (potentially `enum class Sigma1Tags : uint8_t` with some `ToRaw` method to use underlying to get the raw value.

Splitting constexpr and then enums in other cases seems to be more diverging.

Reply:
suggestion integrated.